cmd/kubeadm/app/discovery/file/file.go

```diff
@@ -55,13 +55,23 @@ func ValidateConfigInfo(config *clientcmdapi.Config, clustername string) (*clien
 	// This is the cluster object we've got from the cluster-info kubeconfig file
 	defaultCluster := kubeconfigutil.GetClusterFromKubeConfig(config)
 
+	var caCerts []byte
```

Comment:
maybe something `caCerts := defaultCluster.CertificateAuthorityData`

Reply:
Yeah, I prefer to declare like this.